### PR TITLE
Add more timetable documentation

### PR DESCRIPTION
**Pull request by YoRyan**
- Clarify the file extension for timetable group files
- Add documentation for .weather-or feature

### CODE REVIEW

File: Source/Documentation/Manual/timetable.rst [hunk 1]
```diff
@@ -2205,6 +2228,111 @@ One frame per second relates to a rotation speed of 0.1 degrees per second.
...
+Changing Weather
+================
+
+The current cloud cover, precipitation, and visibility can be varied over the 
+course of the timetable day with weather files. Weather files reside within a 
+special ``WeatherFiles`` subdirectory of the route's folder and they have the 
+file extension ``*.weather-or``. They are selected by the player from the 
+timetable mode menu.
+
+A weather file is a JSON file that consists of a single array, named "Changes,"
```
Comment (Sharpe49) [inline Source/Documentation/Manual/timetable.rst:2242]:
```suggestion
A weather file is a JSON file that consists of a single array, named "Changes", 
```

File: Source/Documentation/Manual/timetable.rst [hunk 2]
```diff
@@ -2205,6 +2228,111 @@ One frame per second relates to a rotation speed of 0.1 degrees per second.
...
+Overcast             number The overcast intensity as a percentage from 0 to 100.
+OvercastVariation    number The variation in overcast intensity as a percentage from 0 to 100.
+OvercastRateOfChange number The rate of change of overcast intensity as a scaling factor from 0 to 1.
+OvercastVisibility   number The resulting visibility, in meters. The value must be in the range from 10000 to 60000. (For lower values, use a Fog event.)
+==================== ====== ===========
+
+"Precipitation" event
+---------------------
+
+A Precipitation event represents a rain spell followed by a clear spell, with
```
Comment (Sharpe49) [inline Source/Documentation/Manual/timetable.rst:2282]: About "spell", do you mean something like a phase?

**Review by Sharpe49 — CHANGES_REQUESTED** [review]

File: Source/Documentation/Manual/timetable.rst (hunk 1 above)
Comment (YoRyan) [reply Source/Documentation/Manual/timetable.rst]: Good catch, thanks.

File: Source/Documentation/Manual/timetable.rst (hunk 2 above)
Comment (YoRyan) [reply Source/Documentation/Manual/timetable.rst:2282]: In English, "spell" also means a period of weather. It's also the term that @Roeterdink used in the weather code.

Comment (Sharpe49) [reply Source/Documentation/Manual/timetable.rst:2282]: Alright, thank you.

**Review by Sharpe49 — APPROVED** [review]

**Pull request merged**